examples/baseline_model.ipynb

```diff
@@ -2,20 +2,158 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "0654593b-3ca7-4526-ad1c-9879688b9773",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>y</th>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>data_iniSE</th>\n",
+       "      <th></th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>2010-01-03</th>\n",
+       "      <td>30.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2010-01-10</th>\n",
+       "      <td>44.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2010-01-17</th>\n",
+       "      <td>46.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2010-01-24</th>\n",
+       "      <td>47.0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2010-01-31</th>\n",
+       "      <td>68.0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "               y\n",
+       "data_iniSE      \n",
+       "2010-01-03  30.0\n",
+       "2010-01-10  44.0\n",
+       "2010-01-17  46.0\n",
+       "2010-01-24  47.0\n",
+       "2010-01-31  68.0"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "disease = 'dengue'\n",
+    "geocode = 3304557 \n",
+    "end_date = date.today().strftime('%Y-%m-%d')\n",
+    "\n",
+    "df = Infodengue.get(per_page =  100,\n",
+    "                            disease =  disease,\n",
+    "                            start = \"2010-01-01\",\n",
+    "                            end = end_date, \n",
+    "                            geocode = geocode)\n",
+    "\n",
+    "df.set_index('data_iniSE', inplace = True )\n",
+    "    \n",
+    "df.index = pd.to_datetime(df.index)\n",
+    "    \n",
+    "df = df.sort_index()\n",
+    "    \n",
+    "df = df.asfreq('W')\n",
```

Comment:
Here I think it is better to use `df = df.resample('W').sum()` there is no need to `ffill` afterwards.